bears/general/LineCountBear.py

```diff
@@ -10,14 +12,19 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            exclude_blank_lines: bool=False):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
 
         :param max_lines_per_file: Number of lines allowed per file.
+        :param exclude_blank_lines: True if blank lines are to be excluded.
         """
         file_length = len(file)
+        blank_lines = len(list(filter(lambda x: re.match(r'^\s*$', x), file)))
```

Comment:
Please move this algorithm into a `_get_blank_line_count` function, just to encapsulate the logic.

The assignment of `blank_lines` should be inside the `if ... :` block below.  It is unused computation if `exclude_blank_lines` is disabled.